x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/Watcher.java

```diff
@@ -602,9 +719,14 @@ static void validAutoCreateIndex(Settings settings, Logger logger) {
             return;
         }
 
-        String errorMessage = LoggerMessageFormat.format("the [action.auto_create_index] setting value [{}] is too" +
-                " restrictive. disable [action.auto_create_index] or set it to " +
-                "[{},{}]", (Object) value, Watch.INDEX, TriggeredWatchStoreField.INDEX_NAME);
+        String errorMessage = LoggerMessageFormat.format(
```

Comment:
We should not need this anymore since these are system indices and we should always be able to create the watches index and triggered watches index.

Reply:
I've removed this code.